From e0c10e632a0b3c9f8b58ecd8e47753bc9e1a2128 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 05 Sep 2021 20:20:45 +0300
Subject: [PATCH] [SW-0] gaudi2: remove sram scrambling empty func

Remove the hl_ram_init() and sram_scrambling_init() function altogether.
Those functions are empty.
The scrambling cannot be performed in Preboot anyway, as it runs from
the SRAM itself.
So remove those placeholders to prevent any confusion.

Change-Id: I056d1c0f08d346fa99d344d86690241fc679d123
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/arch/gaudi2/arc/asic.c b/arch/gaudi2/arc/asic.c
index f942998..45d7d02 100644
--- a/arch/gaudi2/arc/asic.c
+++ b/arch/gaudi2/arc/asic.c
@@ -22,16 +22,6 @@
 	return 1;
 }
 
-/* initialize sram scrambling before u-boot is downloaded */
-void sram_scrambling_init(void)
-{
-	/* TODO */
-}
-
-void hl_ram_init(void)
-{
-	sram_scrambling_init();
-}
 /*
  * FW_SRAM layout:
  * ---------------
